pom.xml

```diff
@@ -163,7 +163,7 @@ flexible messaging model and an intuitive client API.</description>
     <reflections.version>0.10.2</reflections.version>
     <swagger.version>1.6.2</swagger.version>
     <puppycrawl.checkstyle.version>10.14.2</puppycrawl.checkstyle.version>
-    <docker-maven.version>0.43.3</docker-maven.version>
+    <docker-maven.version>0.43.4</docker-maven.version>
```

Comment:
0.44 is most recent. the change made in 0.43.4 is reverted and revisited in 0.44, see https://github.com/fabric8io/docker-maven-plugin/releases